test/cql/lwt_test.result

```diff
@@ -4868,6 +4868,16 @@ update lwt set c = 1 where a in () and b = 1 if c = 1;
 	"message" : "exceptions::invalid_request_exception (IN on the partition key is not supported with conditional updates)",
 	"status" : "error"
 }
+update lwt set c = 1 where a = 1 and b IN (1, 2) if c = 1;
+{
+	"message" : "exceptions::invalid_request_exception (IN on the clustering key columns is not supported with conditional updates)",
+	"status" : "error"
+}
+update lwt set c = 1 where a = 1 and (b) IN ((1), (2)) if c = 1;
+{
+	"message" : "exceptions::invalid_request_exception (IN on the clustering key columns is not supported with conditional updates)",
+	"status" : "error"
+}
 drop table lwt;
 {
 	"status" : "ok"
```

Comment:
I'm afraid the patch is very large, and the explanations are very short. As a result, I don't know how to review it.

Reply:
Broke into two commits and added detailed description into the commit message.